pyproject.toml

```diff
@@ -82,10 +82,9 @@ line-length = 88
 src = ["gridstatus"]
 
 [tool.ruff.lint]
-ignore = []
+ignore = ["E501"]
```

Comment:
why add this to ignore?

Reply:
because we are using black, so black will try the formatting to the best of it's ability, but it falls down on comments and things

> Since you're using Black, you can just disable the Ruff E501 rule; Black will keep your lines under its line-length (as well as it can).

From [first answer here.](https://stackoverflow.com/questions/77396198/how-to-auto-format-e501-line-too-long-89-88-characters-in-pre-commit) 

Mostly I'd like to move to ruff-format (especially if we are using ruff-lint), but that feels like a separate PR. This is a  temporary quality of life don't-make-me-count-characters-even-if-black-can't-figure-out-a-docstring addition.

Reply:
Should we move to only using `ruff`? It doesn't seem like you need both ruff and black.

Reply:
+1 to using only `ruff`. In a past life I've struggled with `ruff` and `black` wanting different things, so sticking with one tool seems good to me.

Reply:
> I can make that change in the next PR.

That would be great, thanks!

Reply:
Tracked in ENG-797